types/browan/TBClimate/meta.json

```diff
@@ -1,23 +1,20 @@
 {
-  "name": "Temperature & Humidity Sensor, TBHH100, Tabs",
+  "name": "Tabs Temperature & Humidity Sensor, TBHH100",
   "version": "1.0.0",
   "manufacturer": "Browan",
   "url": "https://www.browan.com/product/temperature-humidity-sensor/detail",
-  "description": "LoRaWAN® device for measuring the indoor environment ",
+  "description": "The Tabs Temperature & Humidity Sensor is designed for in-home and in-building usage for consumer or facility management applications. ",
   "author": "Akenza AG",
   "firmwareVersion": "V1.0.0",
   "loraDeviceClass": "A",
   "availableSensors": [
     "Temperature",
-    "Humidity",
-    "CO2",
-    "VOC",
-    "Reed"
+    "Humidity"
```

Comment:
At least Co2 and VOC should be included https://core.akenza.io/device/0297eaf160518c02?environment=2997cd32857f846d

Reply:
@J2thatsme this sensor was probably mixed with the IAQ Healthy Home sensor from browan which provides CO2 and VOC. (sensor which you have linked)
The TBHH100 which is the sensor in this case does not provide any CO2 or VOC value.